code/datums/lazy_template.dm

```diff
@@ -7,11 +7,20 @@
 	/// If this is true each load will increment an index keyed to the type and it will load [map_name]_[index]
 	var/list/datum/turf_reservation/reservations = list()
 	var/uses_multiple_allocations = FALSE
+	/// Key to identify this template - used in caching
 	var/key
+	/// Directory of maps to prefix to the filename
 	var/map_dir = "_maps/templates/lazy_templates"
+	/// The filename (without extension) of the map to load
 	var/map_name
+	/// The map width for turf reservation
 	var/map_width
+	/// The map height for turf reservation
 	var/map_height
```

Comment:
these vars are being removed in my latest pr.
```suggestion
```

Reply:
oh nice. i'll have to scrap them from the rest of the vdoms.

Reply:
will this cause any errors if i remove it now?

Reply:
shouldn't, literally not used anywhere lmao

Reply:
i'll wait for it. i assume this PR will still be up well past when yours is merged.

i will also need to change how i'm getting the generated atoms list